src/app/core/services/label.service.ts

```diff
@@ -30,7 +54,7 @@ export class LabelService {
   getAllLabels(userResponse: User): Observable<User> {
```

Comment:
why is `getAllLabels` (which sounds like a query) is also updating labels?

Reply:
Yes, I am currently working on this as part of introducing the `settings.json` in #85 and left it as it is here because #85 requires a major rework of the authentication process.

Reply:
Maybe I'll rename it to `synchronizeRemoteLabels` for now and rework the logic of this function in that PR.